vms/main.tf

```diff
@@ -0,0 +1,36 @@
+variable "deployment_id" {}
+variable "instance_name_suffix" {}
+variable "vm_name_prefix" {}
+variable "latest_al2_linux_ami" {}
```

Comment:
variables should go in separate file called `variables.tf`

Reply:
thanks, moved vars to variables.tf file